playing_game.py

```diff
@@ -0,0 +1,43 @@
+#!/bin/python3
+
+import os
+import sys
+
+#

```

Comment:
What is the problem?

Reply:
Sorry i have confused you. I started providing link on commented section so , i think this is making bit harder. From coming problems, i will update you in the file itself. 
https://www.hackerrank.com/challenges/an-interesting-game-1/problem